templates/vue/src/components/Combobox.vue

```diff
@@ -46,8 +46,9 @@ export default {
       required: true,
     },
     value: {
-      type: [Object, String],
-      required: true,
+      type: [Object, String, null],
```

Comment:
@narendrasss is adding `null` necessary here? I noticed that in the TYDE version of this PR you didn't include this and so was wondering which one is correct (I think since `required` is false, `null` is not necessary to be added in the `types` array, but wanted to confirm)

Reply:
@wynnset Oh yeah I looked it up and it’s not necessary, forgot to update it on the base branch.

Reply:
Np I'll update it thanks